hotel-management/src/constants/routePath.ts

```diff
@@ -0,0 +1,12 @@
+const ROUTE_PATH = {
+  ERROR_PAGE: '*',
+  HOME_PAGE: '/',
+  ROOM_DETAILS_PAGE: '/room/:id',
+  CART_PAGE: '/cart',
+  LOGIN_PAGE: '/login',
+  REGISTER_PAGE: '/register',
+  ADD_ROOM_PAGE: '/add-room',
+  EDIT_ROOM_PAGE: '/edit-room',
+} as const;
```

Comment:
Why we need `as const` here

Reply:
Because these routes cannot be modified by any means, and by adding as const, I can see the literal type of every key in ROUTE_PATH